anitya/db/__init__.py

```diff
@@ -32,4 +32,4 @@
     User,
     ApiToken,
 )
-from .events import set_ecosystem  # noqa: F401
+from .events import set_ecosystem_backend, set_ecosystem_homepage  # noqa: F401
```

Comment:
Is there even a reason to import `set_ecosystem_*` here? AIUI, neither function is used anywhere else in the code, they just work by being decorated with `event.listens_for()`.

Reply:
Probably not, it will be better to remove it.

Reply:
After trying to remove it, I could say that this is needed. Without it the events are not registered and just skipped.

Reply:
I added a comment to this line, so it's clear, why it's needed.

Reply:
AIUI, merely importing the module should be sufficient:

````
# The events module needs to be imported so the handlers are registered.
from . import events  # noqa: F401
````